src/robusta/core/sinks/jira/jira_sink_params.py

```diff
@@ -13,6 +13,10 @@ class JiraSinkParams(SinkBaseParams):
     project_name: str
     project_type_id_override: Optional[int]
     issue_type_id_override: Optional[int]
+    sendResolved: Optional[bool]
+    reopenIssues: Optional[bool]
+    doneStatusName: Optional[str] = "Done"
```

Comment:
these statuses "Done" and "To Do" should be "default" ones in Jira, as I understand from the [docs](https://support.atlassian.com/jira-cloud-administration/docs/what-are-issue-statuses-priorities-and-resolutions/)